packages/atlas/src/components/Text/Text.styles.ts

```diff
@@ -1,12 +1,20 @@
 import { css } from '@emotion/react'
 import styled from '@emotion/styled'
 
-import { cVar } from '@/styles'
+import { cVar, sizes } from '@/styles'
+
+export type Spacing = {
+  left?: number
+  right?: number
+  top?: number
+  bottom?: number
+}
 
 type TextProps = {
   color?: string
   isSecondary?: boolean
   clampAfterLine?: number
+  spacingConfig?: Spacing
```

Comment:
What's the reasoning behind this? I'm not sure how I feel about this change, this way we move styling into the components markup, instead of style files like we do currently

Reply:
We have a lot of `Text` components which have own styled component only because of margin. I thought it would save some work in the future.